docs/UserGuide.md

```diff
@@ -590,6 +590,17 @@ _Note that for the above image, the tag `friends` does not contain sub-tags._
 
 * `view -t t/cs2030 t/cs2040` Shows the details for the tags `cs2030` and `cs2040` in a sequential order.
 
+#### Viewing tag tree (coming soon) - `viewtagtree`
+
+Displays a visual representation of tag-to-tag relationships that currently exist in Athena. This upcoming feature will allow users to easily understand the current tag structure that they have built.
+
+A preview image is shown below for you to understand what this feature will look like.
+![tagtree](images/tagtree-test-tree.png)
+
+Format: `viewtagtree -t`
+* Displays the current tag tree in Athena.
+* For each tag, a summary of contacts directly tagged with it is shown. If there are too many contacts, only the first few will be shown together with a number indicating the number of undisplayed contacts.
+
```

Comment:
it might be better to separate this into another section, since grouping it with existing features may be confusing for the user.

Reply:
done! separated it out. Though it looks slightly awkward now as the upcoming features have to be listed with a `coming soon` as specified by the UG requirements.